includes/class-logger.php

```diff
@@ -45,21 +45,7 @@ class Logger {
 	 *     'debug': Debug-level messages.
 	 */
 	public static function log( $message, $level = 'info' ) {
-		if ( ! self::can_log() ) {
-			return;
-		}
-
-		self::init_logger();
```

Comment:
Now that we're not storing the logger, the `private static $logger` property can be removed, as well as `use WC_Logger`.

Reply:
Done